doc/integrator/ngeo.rst

```diff
@@ -108,3 +108,23 @@ The map configuration will be here:
    If you previously had a CGXP application and want to keep your permalinks
    compatible, the resolutions should be the same as in the previous
    application.
+
+Dynamic.js view
+---------------
+
+To configure the ngeo constance with dynamic or configurable bales we have the dynamic view.
+
+This view is configurable in the vars files in the section ``interfaces_config``.
+The sub section is the interface name, and after we have:
+
+* ``redirect_interface``: interface on witch one we can be redirected if we use the wrong type of device
+  (mobile/desktop).
+* ``do_redirect``: directly to the redirect.
+* ``constants``: Dirrectly define an ``angular`` constant in the vars file.
+* ``routes``: key: constant name, value name of the route witch one we want to have the URL.
```

Comment:
I do not understand.
How I suppose it works: 
``routes``: ngeo constant name: pyramid route name, same as a constant, but interpreted on server side through pyramid request.route_url.

Reply:
Yes :-)

Reply:
"configuration bales" seems stange to me, but I'm not high level english speaker.